tools/Microsoft.StandardsPolice/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "version": "1.0.0-*",
 
+  "compilationOptions": {
```

Comment:
@Eilon @davidfowl we compile build time dependent projects using their own compilation options to separate assembly, thats why if it's not signed we get load exception.